scripts/debug_tools/failure_lib.py

```diff
@@ -0,0 +1,47 @@
+import json
+import os
+
+
+def findPath(string, path_begin):
```

Comment:
Maybe this could be renamed to `findPathEnd`?

Reply:
Well, it finds a path, and returns that too. (On the other hand it returns the path end too.) So I would keep the name.

Reply:
Yeah, but the beginning of the path is the same as the argument and is found by a separate piece of code.